### ISSUE TITLE
[BUG] Adding “Album Artist”, “Composer” or “Disc” tags to opus files fails silently

### ISSUE THREAD

**Issue by vejkse** [ncmpcpp/ncmpcpp#515]
**Describe the bug**
Adding “Album Artist”, “Composer” or “Disc” tags to opus files fails silently.

**To Reproduce**
1. Go to the Tag editor
2. Choose an .opus file
3. Fill all tags and save.
4. After updating the database, “Album Artist”, “Composer” and “Disc” become empty in the tag editor and they are not written in the file metadata (checked with `mediainfo` and `ffprobe`).  No error message is displayed, nor added to `ncmpcpp/error.log`.  (“Comment” also becomes empty in the tag editor, but that’s expected behaviour: #247 and it is written in the file.)

**Expected behavior**
I would expect those tags to be written in the file (and consequently to appear as filled in the tag editor).  Everything works fine with mp3 files and, after adding the same tags “by hand” to the same opus file and updating the database, they do appear as filled in the tag editor:
```
ffmpeg -i track.opus \
  -metadata ALBUMARTIST='Someone' \
  -metadata COMPOSER='Someone Else' \
  -c:a copy \
  track-tagged.opus
```
**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Terminal: Alacritty/bash
 - Version: 0.9.2

**Comment by tibequadorian** [comment]
fixed in #463

**Comment by karel-ferraa** [comment]
I am currently encountering the same issue. The Album Artist, Composer, Performer and Disc tags are not being written. That is, if I write them, save and reopen the editor, those fields will be set to the previous value. I have confirmed that the tags are being read correctly, that is, if I set the tags using another tag editor such as kid3, the tags appear in ncmpcpp.

**Comment by ErtaiBlue** [comment]
This issue does not seem fixed, I am currently encountering the same issue.
The Album Artist, Composer, Performer and Disc tags are not being written. That is, if I write them, save and reopen the editor, those fields will be set to the previous value. I have confirmed that the tags are being read correctly, that is, if I set the tags using another tag editor such as kid3, the tags appear in ncmpcpp.
